### PR TITLE
[CPU] [ARM] JIT HSigmoid: Min & Max

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [ARM] JIT HSigmoid: Min & Max*

### Tickets:
 - *CVS-139331*

### CODE REVIEW

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp [hunk 1]
```diff
@@ -160,6 +160,47 @@ class jit_exp_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const;
 };
 
+class jit_maximum_emitter : public jit_emitter {
```
Comment (xuchen-intel) [inline src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp:163]: Seems we need test coverage about new added max and min emitters.

**Review by xuchen-intel — CHANGES_REQUESTED** [review]

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 2]
```diff
@@ -391,6 +391,82 @@ std::set<std::vector<element::Type>> jit_exp_emitter::get_supported_precisions(c
...
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    TReg src1 = TReg(in_vec_idxs[0]);
+    TReg src2 = TReg(in_vec_idxs[1]);
+    TReg dst = TReg(out_vec_idxs[0]);
+
+    h->fmaxnm(dst.s, src1.s, src2.s);
+}
+
+std::set<std::vector<element::Type>> jit_maximum_emitter::get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
+    return {{element::f32}};
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp:163]: `Minimum` and `Maximum` operation emitters are coverd by tests: `MaxMinLayerTest`, `ConversionLayerTest`, `FakeQuantizeLayerTest` and others. But used inference primitive was not checked. I added new test cases for `EltwiseLayerCPUTest`. Thanks for comment.

**Review by xuchen-intel — APPROVED** [review]

File: src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp [hunk 3]
```diff
@@ -60,7 +60,9 @@ enum EltwiseTypes {
     BITWISE_AND,
     BITWISE_NOT,
     BITWISE_OR,
-    BITWISE_XOR
+    BITWISE_XOR,
+    ELTWISE_MINIMUM,
```
Comment (dmitry-gorokhov) [inline src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp:64]: Min and Max are part of MinMaxOpType type and are used in MaxMinLayerTest. We either need to use MinMaxOpType enum to test ARM JIT functionality or fully unite it with EltwiseTypes (remove MaxMinLayerTest test and use Eltwise test instead)

Comment (eshoguli) [reply src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp]: discussed offline, was removed

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**